Script/Source/Entities/Entity.ts

```diff
@@ -50,6 +53,9 @@ namespace Script {
         resistancesSet? = new Set<SPELL_TYPE>();
         startDirection?: number;
         activeEffects = new Map<SPELL_TYPE, number>();
+        moved: boolean;
+        // @Björn ich denke es wäre einfacher / besser die aktuelle rotation zu speichern
```

Comment:
Warum hast du dich jetzt doch für die Richtung in der Form [x,y] und gegen die Rotation als eine Zahl entschieden?

Und die `@Björn` Kommentare sollten noch weg.